app/jobs/irs_attempts_events_batch_job.rb

```diff
@@ -14,17 +14,32 @@ def perform(timestamp: Time.zone.now - 1.hour, dir_path: './attempts_api_output'
       data: event_values, timestamp: timestamp, public_key: pub_key,
     )
 
-    # write to a file and store on the disk until S3 is setup
-    FileUtils.mkdir_p(dir_path)
+    # Write the file to S3 - Can we skip the file write step? Do we need to write out a temp file?
+    if IdentityConfig.store.irs_attempt_api_bucket_name.nil?
+      # write to a file and store on the disk until S3 is setup
+      FileUtils.mkdir_p(dir_path)
```

Comment:
Let's use `Dir.mktmpdir` (preferably in the block form) for these temporary files so we can have some assurance they're cleaned up after

Reply:
This piece of code is likely going away once S3 inegration is working. We should no longer need a temp file at all.